util.js

```diff
@@ -44,19 +44,21 @@ exports.wait = function () {
 }
 
 const originalValue = exports.originalValue = function (value) {
-  // setting `oldProps` for backward-compatibility with old metadta
-  const oldProps = ['cyphertext', 'private', 'unbox']
-
-  const metaProps = oldProps.concat('meta')
-  const original = value.meta && value.meta.original || {}
-
-  var o = {}
-  for (var key in value) {
-    if (!metaProps.includes(key))
-      o[key] = original[key] || value[key]
+  if (value.meta) {
+    if (value.meta.original) {
+      Object.entries(value.meta.original).forEach(entry => {
```

Comment:
btw, what values get stored here? isn't it just content?

Reply:
Only `content`, but I couldn't detect a performance difference between hardcoding `content` and allowing *any* property of `msg.value` be backed up. For example, if you wanted to transform messages to fix impossible timestamps (e.g. asserted timestamp is in 2060 when received timestamp is in 2018) then it would be important to be able to have the original backed up to `meta.original.timestamp`.